### PR TITLE
Add Trakt notes

### PR DESCRIPTION
**Pull request by fernandog**
Add missing notes

### CODE REVIEW

File: views/config_notifications.mako [hunk 1]
```diff
@@ -1604,6 +1608,7 @@
                                             <input type="checkbox" class="enabler" name="trakt_sync_watchlist" id="trakt_sync_watchlist" ${'checked="checked"' if app.TRAKT_SYNC_WATCHLIST else ''}/>
                                             <p>sync your Medusa show watchlist with your trakt show watchlist (either Show and Episode).</p>
                                             <p>Episode will be added on watch list when wanted or snatched and will be removed when downloaded </p>
+                                            <p><b>Note:</b> By design, Trakt auto remove episode and/or show from watchlist as soon you watched it.</p>
```
Comment (OmgImAlexis) [inline views/config_notifications.mako]: By design, Trakt automatically removes episodes and/or shows from watchlist as soon you have watched them.

File: views/config_notifications.mako [hunk 2]
```diff
@@ -1593,6 +1596,7 @@
                                             <span class="component-desc">
                                                 <input type="checkbox" name="trakt_sync_remove" id="trakt_sync_remove" ${'checked="checked"' if app.TRAKT_SYNC_REMOVE else ''}/>
                                                 <p>Remove an Episode from your Trakt Collection if it is not in your Medusa Library.</p>
+                                                <p><b>Note:</b> Don't enable this setting if you use Trakt addon in Kodi (or other script) that syncs library.</p>
```
Comment (OmgImAlexis) [inline views/config_notifications.mako]: Don't enable this setting if you use the Trakt addon for Kodi or any other script that syncs your library.

File: views/config_notifications.mako [hunk 3]
```diff
@@ -1582,7 +1582,10 @@
                                         <span class="component-title">Sync libraries</span>
                                         <span class="component-desc">
                                             <input type="checkbox" class="enabler" name="trakt_sync" id="trakt_sync" ${'checked="checked"' if app.TRAKT_SYNC else ''}/>
-                                            <p>sync your Medusa show library with your trakt show library.</p>
+                                            <p>sync your Medusa show library(collection) with your trakt show library.</p>
```
Comment (OmgImAlexis) [inline views/config_notifications.mako]:
```
<p>Sync your Medusa show library with your Trakt collection.</p>
<p><b>Note:</b> Don't enable this setting if you use the Trakt addon for Kodi or any other script that syncs your library.</p>
<p>Kodi detects that the episode was deleted and removes from collection which causes Medusa to re-add it. This causes a loop between Medusa and Kodi adding and deleting the episode.</p>
```

**Comment by fernandog** [comment]
@OmgImAlexis thanks! commited

**Pull request merged**